src/delivery-options/config/isPastCutoff.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} time
+ *
+ * @returns {boolean}
+ */
+export function isPastCutoff(time) {
+  const [hour, minutes] = time.split(':');
+  const date = new Date();
+
+  const hourHasPassed = Number(hour) <= date.getHours();
+  const hourIsSame = Number(hour) === date.getHours();
+  const minutesHavePassed = Number(minutes) <= date.getMinutes();
+
+  return hourHasPassed && (hourIsSame ? minutesHavePassed : true);
```

Comment:
Die ternary in de return vind ik persoonlijk viezer dan een mixed && en ||

Alternatieve sugestie voor dit block:

```suggestion
  time  = time.padStart(5,0);
  const date = new Date();
  currentTime = `${date.getHours()}`.padStart(2, 0) + ':' + `${date.getHours()}`.padStart(2, 0);
  return time < currentTime`
```

Reply:
prima, vermoed dat ik de het > had moeten zijn op het einde, maar met deze oplossing ben ik ook happy.

Reply:
Bij nader inzien moest de tweede getHours een getMinutes zijn